### PR TITLE
Dev chains fixes

### LINKED ISSUE #179: mosaic start dev-origin fails
Below command fails because `/Users/user/.mosaic/dev-origin` folder doesn't exist. 
./mosaic start dev-origin
info: starting parity container {"service":"mosaic-chains","chain":"dev-origin","timestamp":"2019-10-14T08:02:12.137Z"}
info: /Users/user/.mosaic/dev-origin/origin-parity does not exist; initializing {"service":"mosaic-chains","chain":"dev-origin","timestamp":"2019-10-14T08:02:12.140Z"}
(node:23616) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, mkdir '/Users/user/.mosaic/dev-origin/origin-parity'

### PR DESCRIPTION
**Pull request by sarvesh-ost**
fixes #179 

Fixes bugs and published new docker image with tag 1.0.0 which contains latest mosaic config and gateway config.

### CODE REVIEW

File: src/Node/ParityNode.ts [hunk 1]
```diff
@@ -23,6 +23,7 @@ export default class ParityNode extends Node {
       args = args.concat('--rm');
     }
 
+    console.log('this.containerName  ',this.containerName);
```
Comment (abhayks1) [inline src/Node/ParityNode.ts]:
```suggestion
```

File: src/Node/ParityNode.ts [hunk 2]
```diff
@@ -72,7 +73,7 @@ export default class ParityNode extends Node {
         '--password', '/home/parity/password.txt',
       ]);
     }
-
+    console.log('args  ',args);
```
Comment (abhayks1) [inline src/Node/ParityNode.ts]:
```suggestion
```

File: src/bin/mosaic-start.ts [hunk 3]
```diff
@@ -66,58 +66,64 @@ mosaic
...
+      node.start();
 
-    if (!optionInput.withoutGraphNode) {
-      const graphDescription: GraphDescription = GraphOptions.parseOptions(optionInput, chainInput);
-      // reuse params from node start command
-      graphDescription.mosaicDir = mosaicDir;
-      graphDescription.ethereumRpcPort = rpcPort;
-      graphDescription.ethereumClient = nodeDescription.client;
+      if (!optionInput.withoutGraphNode) {
+        const graphDescription: GraphDescription = GraphOptions.parseOptions(optionInput, chainInput);
```
Comment (abhayks1) [inline src/bin/mosaic-start.ts]: There are linting issues in this file. on line 108 and 118.

File: test_integration/mosaic-create/Utils.ts [hunk 4]
```diff
@@ -23,7 +23,7 @@ export default class Utils {
         '8546:8546',
         '--name',
         'integration_test_origin',
-        'mosaicdao/dev-chains',
+        'mosaicdao/dev-chains:1.0.0',
```
Comment (abhayks1) [inline test_integration/mosaic-create/Utils.ts]: Do we always need to specify docker version?
Shouldn't dev chain use latest published docker. Facilitator can access specific npm version of mosaic chains.

Comment (sarvesh-ost) [reply test_integration/mosaic-create/Utils.ts]: Yes, Because given version of mosaic-chains will be compatible with specific version of dev-chain.

**Pull request merged**